recipes/domclust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: domclust 
+  version: "1.2.7l"
+
+about:
+  home: http://mbgd.genome.ad.jp/domclust/
+  license: 
```

Comment:
If there is no license you are not allowed to do anything with it :(
Can you ask the authors?